tests/phpunit/tests/media.php

```diff
@@ -3162,6 +3162,31 @@ public function test_wp_img_tag_add_loading_attr_opt_out() {
 		$this->assertStringNotContainsString( ' loading=', $img );
 	}
 
+
+
+	/**
+	 * Test that decoding="async" is not applied to img tags with single quotes.
+	 *
+	 * @ticket 56969
+	 */
+	public function test_wp_img_tag_add_decoding_attr_with_single_quotes() {
+		$img = "<img src='example.png' alt=' width='300' height='225' />";
```

Comment:
Nit-pick, but the HTML here is broken. Doesn't matter for the test but probably good to use a working "realistic" example.
```suggestion
		$img = "<img src='example.png' alt='' width='300' height='225' />";
```